### PR TITLE
wis2box-api for wis2box-api-1.0b7

### PR DESCRIPTION
**Pull request by maaikelimper**

### CODE REVIEW

File: docker_compose_test/tests/integration/test_wis2box.py [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    # create the expected message
+    expected_message = {
+        'channel': expected_response['data_items'][0]['channel'],
+        'filename': expected_response['data_items'][0]['filename'],
+        'data': expected_response['data_items'][0]['data'],
+        '_meta': expected_response['data_items'][0]['_meta']
+    }
+
+    # compare the received message with the expected message
+    assert message['channel'] == expected_message['channel']
```
Comment (tomkralidis) [inline docker_compose_test/tests/integration/test_wis2box.py:121]:
```python
    for key in ['channel', 'filename', 'data', '_meta']:
        assert message[key] == expected_message[key]
```

**Review by tomkralidis — CHANGES_REQUESTED** [review]
Some minor changes requests.  Overall comment to order imports as follows;

- standard library
- 3rd party packages
- local modules

```python
import os
import sys

import paho.mqtt
import requests

import local.foo_module
import local.z_module
```

File: docker_compose_test/tests/integration/test_wis2box.py (hunk 1 above)
Comment (maaikelimper) [reply docker_compose_test/tests/integration/test_wis2box.py]: ok

File: docker_compose_test/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+services:
+  wis2box-api:
+    container_name: wis2box-api-test
+    env_file:
+      - test.env
+    build:
+      context: ./../
+      dockerfile: Dockerfile
```
[the remark this answers is not in the document]
Comment (maaikelimper) [reply docker_compose_test/docker-compose.yml]: nope, removed

File: docker/pygeoapi-config.yml [hunk 3]
```diff
@@ -14,6 +14,10 @@ server:
         attribution: ${WIS2BOX_BASEMAP_ATTRIBUTION}
     templates:
         path: /pygeoapi/pygeoapi/templates
+    manager:
```
[the remark this answers is not in the document]
Comment (maaikelimper) [reply docker/pygeoapi-config.yml:17]: I added a cronjob removing successful jobs after one hour

**Pull request merged**